crates/iota-keys/src/keystore.rs

```diff
@@ -40,11 +42,10 @@ pub enum Keystore {
 
 #[enum_dispatch]
 pub trait AccountKeystore: Send + Sync {
-    fn add_key(&mut self, alias: Option<String>, keypair: IotaKeyPair)
-    -> Result<(), anyhow::Error>;
+    fn add_key(&mut self, alias: Option<String>, key: StoredKey) -> Result<(), anyhow::Error>;
```

Comment:
If you take a `impl Into<StoredKey>` instead it would be backwards compatible, you wouldn't need to sprinkle `.into()` everywhere, and it would be generally easier to use